insights/parsers/getcert_list.py

```diff
@@ -0,0 +1,138 @@
+"""
+Cert_List - command ``getcert list``
+====================================
+
+"""
+
+from insights.core import Parser
+from insights.parsers import ParseException, keyword_search
+from insights.core.plugins import parser
+
+
+@parser('getcert_list')
+class Cert_List(Parser):
```

Comment:
@PaulWay normally we would use just camelcase for class names, `CertList` instead of `Cert_List`.